#!/usr/bin/python3

def raise_exceptions():
    a = 8
    b = '8'
    result = a + b
